themes/vocabulary_theme/templates/layout.html

```diff
@@ -8,6 +8,9 @@
 <link rel="apple-touch-icon" sizes="180x180" href="{{ '/static/vocabulary/favicon/apple-touch-icon.png' |url}}" />
 <link rel="stylesheet" href="{{ '/static/css/style.css'|url }}">
 <link rel="stylesheet" href="{{ get_pygments_stylesheet()|url }}">
+<link rel="stylesheet" href="https://cdn.datatables.net/2.2.1/css/dataTables.dataTables.css" />
+<script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
```

Comment:
for both the jquery and datatables js, we should localize this to the repository with a vendor directory (see how vocabulary handles the normalize css file), and not pull in via a cdn.